test/clocksi_SUITE.erl

```diff
@@ -130,7 +132,7 @@ clocksi_test1(Config) ->
                       {read, {Key1, Type}}]]),
     ?assertMatch({ok, _}, Result2),
     {ok, {_, ReadSet2, _}}=Result2,
-    ?assertMatch([0,0], ReadSet2),
+    ?assertMatch([0,1], ReadSet2),
```

Comment:
Why is this change in the test necessary?
We should not fix tests and modify code in the same pull request unless there is a good reason.